lib/screens/home/pages/relation/relation_page.dart

```diff
@@ -49,7 +52,47 @@ class _RelationPageState extends State<RelationPage> {
 
               if (state is RelationPageFailure) {
                 return Center(
-                  child: Text(state.message),
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Text(state.message),
+                      SizedBox(
+                        height: MediaQuery.of(context).size.height / 40,
+                      ),
+                      Container(
+                        height: MediaQuery.of(context).size.height / 17,
+                        width: MediaQuery.of(context).size.width * 0.47,
+                        child: RaisedButton(
+                          color: Theme.of(context).accentColor,
+                          onPressed: () {
+                            //ignore: close_sinks
```

Comment:
what is this about? the ignore statement?

Reply:
@isabelcosta that currently is a bug (false positive) in Flutter itself. So to remove the warning message we comment the ignore part. You can read more about it [here](https://github.com/felangel/bloc/issues/587)